documentation/docs/30-advanced/20-hooks.md

```diff
@@ -164,6 +164,7 @@ const Sentry: any;
 
 // @filename: index.js
 // ---cut---
+import * as Sentry from '@sentry/svelte';
```

Comment:
For the server hook, we should use `@sentry/node`, as `@sentry/svelte` only works on the browser-side.

```suggestion
import * as Sentry from '@sentry/node';
```